README.md

```diff
@@ -47,6 +47,10 @@ Prebuilt binaries for macOS and Windows can also be downloaded from the
 For everyone else, the detailed instructions to install Alacritty can be found
 [here](INSTALL.md).
 
+Since Alacritty uses OpenGL to render the terminal content on all platforms, it
+is currently not possible to run Alacritty without at support for at least
```

Comment:
:P

```suggestion
is currently not possible to run Alacritty without support for at least
```

Reply:
God damn it, looks like my force push didn't go through. Thanks.